### DISCUSSION TITLE
Provieder 6591 freetzbar?

### DISCUSSION THREAD

**Discussion by Conan179** [Freetz-NG/freetz-ng#652] [Allgemein]
Hallo erst mal.
Ich hab da mal eine frage, kann ich meine Provider 6591 freetzen? sie ist nicht mehr die aktive Box und soll als Reapter genutzt werden.

**Comment by roland3939** [comment]
---> Ich hab da mal eine frage, kann ich meine Provider 6591 freetzen? <---
Ich habe da Mal eine Antwort: Ja !
